src/inotify.rs

```diff
@@ -405,7 +405,7 @@ impl Inotify {
     pub fn event_stream<T>(&mut self, buffer: T)
         -> EventStream<T>
     where
-        T: AsMut<[u8]> + AsRef<[u8]>,
+        T: AsMut<[u8]> + AsRef<[u8]> + Unpin,
```

Comment:
I'm a bit concerned about the `Unpin`, as it makes the API less flexible. I think right now one can pass an array, but with the `Unpin`, you'd either need a slice (possibly causing lifetime issues) or a heap allocation.

I'm not sure if it can be avoided and if that's a big deal in practice. I just figured I'd point out my concerns, in case you have something informative to say :-)

Reply:
Arrays *are* `Unpin`. But your general point is valid.

Reply:
Interesting, I didn't think they were!